rmgpy/ml/estimator.py

```diff
@@ -28,17 +25,24 @@
 #                                                                             #
 ###############################################################################
 
+from argparse import Namespace
```

Comment:
Put from imports below imports

Reply:
Is that a PEP-8 rule or an RMG rule? I thought imports were supposed to be alphabetical, but I might be wrong.

Reply:
Great question! I thought it was in PEP-8 but apparently not. It seems that it's just the default PyCharm style.

I like grouping from and direct imports separately, but I would want to get everyone's opinions before making it an RMG rule.

Reply:
So in any case we want the following order for each module:

1. standard library imports
2. related third party imports
3. local application/library specific imports

And then I think I agree with you that it makes sense to first have normal imports and then from imports and imports should be alphabetical within each group.